.github/workflows/linkinator.yml

```diff
@@ -0,0 +1,27 @@
+name: Linkinator
+
+on:
+  schedule:
+    - cron:  '0 0 * * 1'
+
+env:
+  GATSBY_DEFAULT_SITE_URL: https://neon.tech
+jobs:
+  linkinator:
+    name: "Checks broken links"
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+      - uses: actions/checkout@v2
+      - uses: JustinBeckwith/linkinator-action@v1
+        with:
+          paths: ${{ env.GATSBY_DEFAULT_SITE_URL }}
+          concurrency: 100
+          recurse: true
+          retry: true
+          skip: ".*(\\.js|\\.css|\\.woff2)$"
+          linksToSkip: >-
+            linkedin.com,
+            googleapis.com
```

Comment:
Very curious to know why do we have exactly those 2 :)

Reply:
The linkedin has protection from crawlers, so it's always returns `0`